### ISSUE TITLE
Add "Show more" button

### ISSUE THREAD

**Issue by Eldenroot** [dragonexpert/recentthreads#37]
One used on my board had this request (idea). Add a button "Show more" to the bottom of the recenthread box. It would load (show on a new page) more latest post than the amount set in ACP.

**Comment by dragonexpert** [comment]
So basically you want a new page that sorts all threads and displays more than the amount set in the ACP?  Would said page need refreshing or would doing pagination be better?

**Comment by Eldenroot** [comment]
Maybe just to show 30 entries would be enough :)

**Comment by dragonexpert** [comment]
Alright, I'll see what I can do tomorrow.

**Comment by dragonexpert** [comment]
On my local copy I am using misc.php?action=recent_threads.  Although no button is currently implemented.  Won't do a pull request until I make it an actual template instead of hard coded html though.

**Comment by Eldenroot** [comment]
OK, nice :)

**Comment by Eldenroot** [comment]
@dragonexpert - hi, any progress? thank you!

**Comment by dragonexpert** [comment]
First priority is getting the broken avatar url patch tested.  Second is to look at the thread read and if I can do anything about it ( not sure because it is complicated ), then this.

**Comment by Eldenroot** [comment]
@dragonexpert - for me, the highest priority is this issue because it is bugged somehow - #45 , the rest all only "improvements".

**Comment by dragonexpert** [comment]
Not a button, but this latest code now support misc.php?action=recent_threads to get more threads.  I have 30 hard-coded, but you can edit it by finding the function recentthread_page in /inc/plugins/recentthreads/hooks.php near the bottom.

**Comment by Eldenroot** [comment]
Great, I will check on friday

**Comment by dragonexpert** [comment]
Managed to get Mass Moderation working on my copy for misc.php?action=recent_threads.  In doing it, I also discovered a bug in the current code which doesn't show the lock status.  I got that resolved as well on my copy.  I couldn't get the MyBB built-in code to work for mass moderation so unfortunately the template will contain a fair bit of Javascript.  I have to be to work at 5am tomorrow and it is getting late so the code here will be updated after work tomorrow.
